cuebot/src/main/java/com/imageworks/spcue/rqd/RqdClientGrpc.java

```diff
@@ -130,7 +135,11 @@ public void lockHost(HostInterface host) {
         RqdStaticLockAllRequest request = RqdStaticLockAllRequest.newBuilder().build();
 
         try {
-            getStub(host.getName()).lockAll(request);
+            String hostName = host.getName();
+            if (env.getProperty("grpc.use_fqdn", Boolean.class, false)) {
+                hostName = host.getFqdn();   
+            } 
```

Comment:
This chunk of code is repeated a lot -- can we extract this logic into a function instead? We could add a `getHostname` function, and a unit test for it as well.

Reply:
I've added private functions a the top of this file to handle this logic now